Documentation/getting-started.md

```diff
@@ -0,0 +1,145 @@
+# Getting Started
+
+If you would like to contribute, but don't know how to start, this page is for
+you. This guide will walk you through running Oreboot locally using QEMU
+emulation and explain the high level architecture / structure of the project.
+It also assumes the use of Debian like system.
+
+## Set up the environment
+
+The first step, is to install all dependencies and ensure the project can be
+started locally (see `README.md` and `azure-pipelines.yml` for possibly more
+up-to-date instructions.
+
+```shell
+git clone git://github.com/oreboot/oreboot
+cd oreboot
+make debiansysprepare
+make firsttime
+```
+
+Next, we need to install QEMU - generic machine emulator that will allow us to
+emulate running oreboot without the need of custom hardware. Below commands
+might be a little outdated, so please look at `azure-pipelines.yml` file for the
+actual commands that are used in CI.
+
+```shell
+# Go to a directory where QEMU will be fetched
+cd ~/
+# Fetch and build QEMU
+git clone https://github.com/qemu/qemu && cd qemu && git checkout <put tag from azure-pipelines.yml>
+mkdir build-riscv64 && cd build-riscv64
+../configure --target-list=riscv64-softmmu
+make -j$(nproc)
+# Make the binaries available in $PATH, e.g.:
+sudo ln -s $PWD/riscv64-softmmu/qemu-system-riscv64 /usr/bin/
+sudo ln -s $PWD/qemu-img /usr/bin/
```

Comment:
I would suggest symlinking to `/usr/local/bin/` to avoid possible collisions when people install QEMU through their package manager as well.

Reply:
Good point, thanks!